app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,68 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      erb :"/tweets/index"
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :"/tweets/new"
+    else
+      redirect to "/login"
+    end
+  end
+
+  post '/tweets' do
+    if !params[:content].empty?
+      @tweet = current_user.tweets.create(params)
```

Comment:
an instance variable is not necessary here as you will not be using @tweet in the .erb file. 

`tweet = current_user.tweets.create(params)` will be sufficient

and on line 22 you could just use 

```ruby
redirect to "/tweets/#{tweet.id}"
```